common/version/version.go

```diff
@@ -5,7 +5,7 @@ import (
 	"runtime/debug"
 )
 
-var tag = "v4.4.99"
+var tag = "v4.4.100"
```

Comment:
How about increasing the minor version

Reply:
good suggestion.

Reply:
changed in [dced0c6](https://github.com/scroll-tech/scroll/pull/1613/commits/dced0c6a821e757abdbb33486b5f330f84a148ec).